dataland-frontend/src/components/pages/SearchCompaniesForFrameworkData.vue

```diff
@@ -110,9 +151,40 @@ export default defineComponent({
         this.searchBarToggled = false;
       }
     },
+    currentFilteredFrameworks: {
+      handler() {
+        this.updateCombinedFilterIfRequired();
+      },
+      deep: true,
+    },
+    currentFilteredCountryCodes: {
+      handler() {
+        this.updateCombinedFilterIfRequired();
+      },
+      deep: true,
+    },
+    currentFilteredSectors: {
+      handler() {
+        this.updateCombinedFilterIfRequired();
+      },
+      deep: true,
+    },
   },
+  computed: {
+    currentlyVisiblePageText(): string {
+      const totalSearchResults = this.resultsArray.length;
+
+      if (totalSearchResults <= 0) return "No results";
 
+      const startIndex = this.firstShownRow;
+      const endIndex = startIndex + 99 >= totalSearchResults ? totalSearchResults - 1 : startIndex + 99;
```

Comment:
The 100 results per page limit should not appear here as "magic number".
I suggest setting it as variable and also use it for the paginator config, which is set to 100 currently

Reply:
Start fixing this now

Reply:
FIxed.   The rowsPerPage is now given by the parent component as variable and passed down to the search results component